samples/metaapp/src/main/java/com/sourcepointmeta/metaapp/tv/properties/PropertyCardView.kt

```diff
@@ -23,6 +23,7 @@ fun PropertyCardView.bind(item: PropertyDTO) {
     account_id.text = "${item.accountId}"
     gdpr_icon.setImageResource(getResStatusResIcon(item.gdprEnabled))
     ccpa_icon.setImageResource(getResStatusResIcon(item.ccpaEnabled))
+    ccpa_icon.setImageResource(getResStatusResIcon(item.ccpaEnabled))
```

Comment:
why did you change the image resource of `ccpa_icon` 2 times? I think it should be something like `usnat_icon`

Reply:
good observation 👍🏻